src/extension/php_objects.c

```diff
@@ -14,6 +14,15 @@ static zend_llist _function_entry_arrays;
 
 static void _unregister_functions(void *zfe_arr_vp);
 
+typedef struct _dd_registered_entries {
+    char *name;
```

Comment:
These should be `zend_string` so you don't need to compute the length later.

Reply:
I disagree on this one. There is only one use of these 2 vars and once strlen on each. Zend_string would use more memory without any performance benefit. I may be missing something, if so pls, let me know 🙏

Reply:
Ok! Now I get it 👏 Done